polaris-react/src/components/CheckableButton/CheckableButton.scss

```diff
@@ -19,6 +19,11 @@
   border-radius: var(--p-border-radius-1);
   width: auto;
 
+  #{$se23} & {
+    min-height: 24px;
+    min-width: 24px;
+  }
+
```

Comment:
I think this is causing the element to not be centered. Is it needed? 

<img width="694" alt="Screenshot 2023-06-09 at 4 25 35 PM" src="https://github.com/Shopify/polaris/assets/20652326/09e65391-b222-40cd-ade0-bc1287ae49f9">

Reply:
Fixed in https://github.com/Shopify/polaris/pull/9453 👍